App/index.js

```diff
@@ -0,0 +1,142 @@
+const verde = document.getElementById('verde');
+const rojo = document.getElementById('rojo');
+const amarillo = document.getElementById('amarillo');
+const azul = document.getElementById('azul');
+
+const btnIniciar = document.getElementById('btnIniciar');
+
+const ULTIMO_NIVEL = 10;
+class Juego {
+  constructor() {
+    this.iniciarJuego = this.iniciarJuego.bind(this);
+    this.iniciarJuego();
+    this.generarSecuencia();
+    setTimeout(this.pasarDeNivel, 700);
+  }
+
+  iniciarJuego() {
+    this.elegirColor = this.elegirColor.bind(this);
+    this.pasarDeNivel = this.pasarDeNivel.bind(this);
+    this.toggleBtnIniciar();
+    this.nivel = 1;
+    this.colores = {
+      verde,
+      rojo,
+      amarillo,
+      azul,
+    };
+  }
+
+  toggleBtnIniciar() {
+    btnIniciar.classList.contains('ocultar')
+      ? btnIniciar.classList.remove('ocultar')
+      : btnIniciar.classList.add('ocultar');
+  }
+
+  generarSecuencia() {
+    this.secuencia = new Array(ULTIMO_NIVEL)
+      .fill(0)
+      .map(() => Math.floor(Math.random() * 4));
+  }
+
+  pasarDeNivel() {
+    this.subNivel = 0;
+    this.iluminarSecuencia();
+    this.agregarClickColors();
+  }
+
+  convertirNumEnCol(numero) {
```

Comment:
Creo que podríamos colocar nombres más específicos para mejorar la legibilidad

```suggestion
  convertirNumeroEnColor(numero) {
```

Reply:
Cambio realizado